### PR TITLE
remove phdi

### PR DESCRIPTION
**Pull request by DanPaseltiner**
# PULL REQUEST

## Summary
Removing `phdi`

## Related Issue
Fixes #

## Acceptance Criteria
Please copy the acceptance criteria from your ticket and paste it here for your reviewer(s)

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Review by fzhao99 — APPROVED** [review]
def wins the PR for best ratio of "addition by subtraction"

File: package.json
Comment (robertandremitchell) [inline package.json:1]: we might want to save this until we remove `turbo`

File: package-lock.json
Comment (robertandremitchell) [inline package-lock.json:1]: we might want to save this until we remove `turbo`

**Review by robertandremitchell — COMMENTED** [review]
we might want to save the packages until we disentangle turbo from the `tefca-viewer`, but otherwise, looks good

File: package.json
Comment (DanPaseltiner) [reply package.json:1]: good call. I restored those files

**Pull request merged**